.github/workflows/main.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       - name: Install webform_civicrm
         run: |
           cd ~/drupal
-          COMPOSER_MEMORY_LIMIT=-1 composer require drupal/webform_civicrm *@dev
+          COMPOSER_MEMORY_LIMIT=-1 composer require drupal/webform_civicrm:dev-5.x@dev
       - name: Install token
```

Comment:
@demeritcowboy - I just checked some other Matt github actions and it does look like -> this is correct:
`           COMPOSER_MEMORY_LIMIT=-1 composer require drupal/webform_civicrm *@dev`

civicrm entity e.g. has:
`          COMPOSER_MEMORY_LIMIT=-1 composer require drupal/civicrm_entity *@dev`